Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:2.7-alpine
+MAINTAINER Andrew Ekstedt <ekstedta@oregonstate.edu>
```

Comment:
Could you use [LABEL](https://docs.docker.com/engine/reference/builder/#label) here instead? https://docs.docker.com/engine/reference/builder/#maintainer-deprecated